packages/events/src/FederatedEventMap.ts

```diff
@@ -4,12 +4,17 @@ import type { FederatedWheelEvent } from './FederatedWheelEvent';
 export type FederatedEventMap = {
     click: FederatedPointerEvent;
     mousedown: FederatedPointerEvent;
+    mouseenter: FederatedPointerEvent
+    mouseleave: FederatedPointerEvent
```

Comment:
```suggestion
    mouseenter: FederatedPointerEvent;
    mouseleave: FederatedPointerEvent;
```

Reply:
Good catch! Must have run the linter wrong because it didn't find this 😬